src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs

```diff
@@ -13,19 +13,16 @@ public class Issue3262 : TestContentPage // or TestFlyoutPage, etc ...
 
 		protected override void Init()
```

Comment:
The build is failing:
```
/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs(47,16): error CS0246: The type or namespace name 'Windows' could not be found (are you missing a using directive or an assembly reference?) [/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-maccatalyst]
/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs(47,5): error CS1929: 'IPlatformElementConfiguration<Windows, WebView>' does not contain a definition for 'SetIsJavaScriptAlertEnabled' and the best extension method overload 'WebView.SetIsJavaScriptAlertEnabled(IPlatformElementConfiguration<Windows, WebView>, bool)' requires a receiver of type 'Microsoft.Maui.Controls.IPlatformElementConfiguration<Microsoft.Maui.Controls.PlatformConfiguration.Windows, Microsoft.Maui.Controls.WebView>' [/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-maccatalyst]
/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs(47,16): error CS0246: The type or namespace name 'Windows' could not be found (are you missing a using directive or an assembly reference?) [/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-tizen]
/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs(47,5): error CS1929: 'IPlatformElementConfiguration<Windows, WebView>' does not contain a definition for 'SetIsJavaScriptAlertEnabled' and the best extension method overload 'WebView.SetIsJavaScriptAlertEnabled(IPlatformElementConfiguration<Windows, WebView>, bool)' requires a receiver of type 'Microsoft.Maui.Controls.IPlatformElementConfiguration<Microsoft.Maui.Controls.PlatformConfiguration.Windows, Microsoft.Maui.Controls.WebView>' [/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-tizen]
/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs(47,16): error CS0246: The type or namespace name 'Windows' could not be found (are you missing a using directive or an assembly reference?) [/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-ios]
/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs(47,5): error CS1929: 'IPlatformElementConfiguration<Windows, WebView>' does not contain a definition for 'SetIsJavaScriptAlertEnabled' and the best extension method overload 'WebView.SetIsJavaScriptAlertEnabled(IPlatformElementConfiguration<Windows, WebView>, bool)' requires a receiver of type 'Microsoft.Maui.Controls.IPlatformElementConfiguration<Microsoft.Maui.Controls.PlatformConfiguration.Windows, Microsoft.Maui.Controls.WebView>' [/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-ios]
/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs(47,16): error CS0246: The type or namespace name 'Windows' could not be found (are you missing a using directive or an assembly reference?) [/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-android]
/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs(47,5): error CS1929: 'IPlatformElementConfiguration<Windows, WebView>' does not contain a definition for 'SetIsJavaScriptAlertEnabled' and the best extension method overload 'WebView.SetIsJavaScriptAlertEnabled(IPlatformElementConfiguration<Windows, WebView>, bool)' requires a receiver of type 'Microsoft.Maui.Controls.IPlatformElementConfiguration<Microsoft.Maui.Controls.PlatformConfiguration.Windows, Microsoft.Maui.Controls.WebView>' [/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-android]
    0 Warning(s)
    8 Error(s)
```
Could you review it?

Reply:
Should work now